_includes/overlay_nav.html

```diff
@@ -1,16 +1,35 @@
-<button id="main-nav-toggle" onClick="handleNav()">
+<a href="#first-content" class="skip-to-main-content-link">Skip to main content</a>
+<button id="main-nav-toggle" onClick="handleNav()" aria-label="Menu" aria-controls="main-nav">
```

Comment:
Not super related to your code, but generally I'd suggest attaching listeners in Javascript instead of in HTML. That is: get rid of `onClick` and use `document.getElementById("main-nav-toggle").addEventListener("click", ...` instead.

Keeping the HTML clean means it is easier to read and evaluate. So let's try to make sure Javascript is only ever started in  `<script>` tags.